### PR TITLE
Make --target default to ${user.dir}

### PR DESCRIPTION
**Pull request by dwalluck**

### CODE REVIEW

File: cli/src/main/java/org/jboss/gm/cli/Main.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
     @Option(names = "--trace", description = "Enable trace.")
     private boolean trace;
 
-    @Option(names = { "-t", "--target" }, required = true, description = "Target Gradle directory.")
+    @Option(names = { "-t", "--target" }, required = true, description = "Target Gradle directory.", defaultValue = ".")
```
Comment (rnc) [inline cli/src/main/java/org/jboss/gm/cli/Main.java:68]: I wonder if we should match it to PME https://github.com/release-engineering/pom-manipulation-ext/blob/master/cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java#L109 (and update the docs of course)

Comment (dwalluck) [reply cli/src/main/java/org/jboss/gm/cli/Main.java]: I switched from "." to "${user.dir} since PME uses that.

Comment (dwalluck) [reply cli/src/main/java/org/jboss/gm/cli/Main.java]: Added to help output "default: ${DEFAULT-VALUE}", but I kind of don't like it as that makes it "dynamic".

Comment (dwalluck) [reply cli/src/main/java/org/jboss/gm/cli/Main.java]: I checked the gh-pages branch. Aside from maybe changing the help output, there is really no mention of `--target`.

File: cli/src/main/java/org/jboss/gm/cli/Main.java [hunk 2]
```diff
@@ -65,7 +65,9 @@
     @Option(names = "--trace", description = "Enable trace.")
     private boolean trace;
 
-    @Option(names = { "-t", "--target" }, required = true, description = "Target Gradle directory.")
+    @Option(names = { "-t", "--target" },
+            description = "Target Gradle directory (default: ${DEFAULT-VALUE}).",
+            defaultValue = "${sys:user.dir}")
```
Comment (rnc) [inline cli/src/main/java/org/jboss/gm/cli/Main.java:70]: Nice!

**Pull request merged**